.github/renovate.json

```diff
@@ -8,8 +8,27 @@
     {
       "description": "Group all dependencies for Java sample",
       "matchPaths": ["examples/java/**"],
-      "groupName": "Java sample dependencies"
-    }
+      "groupName": "Java sample dependencies",
+      "schedule" : ["every weekday"]
```

Comment:
Would it be better to do these like once a week or maybe once a month to avoid overwhelming the git history?

Also, have you considered combining all dependencies in the examples folder together?  

cc @enocom

Reply:
We were just thinking about doing these weekly. Further combining all languages seems like another good idea.